### PR TITLE
Telecomms monitor is speedier now

### PR DESCRIPTION
**Pull request by Crazylemon64**
Full speed ahead
:cl:Crazylemon
fix: Telecomms monitor is now infused with fastitude
/:cl:

### CODE REVIEW

**Comment by Krausus** [comment]
This won't make nearly as much of a difference as getting rid of the mob creation+deletion for every single message would.

**Comment by Crazylemon64** [comment]
OH RIGHT THAT STUPID THING NOW I REMEMBER

1 sec

File: code/game/machinery/telecomms/logbrowser.dm [hunk 1]
```diff
@@ -75,37 +76,34 @@
...
 
-						if(ishuman(M) || isbrain(M))
-							var/mob/living/carbon/human/H = M
-							race = "[H.species.name]"
+						if(ispathhuman(mobtype) || ispathbrain(mobtype))
+							race = "Human" // The initializing thing wouldn't work anyways since it just kept the path, and kept no species data
 
 
-						else if(issmall(M))
+						else if(ispath(mobtype, /mob/living/carbon/human/monkey)) // I am aware this will only work for always-monkeys, and not
```
Comment (Fox-McCloud) [inline code/game/machinery/telecomms/logbrowser.dm]: can't you abuse `initial` to still utilize `issmall`?

Comment (Crazylemon64) [inline code/game/machinery/telecomms/logbrowser.dm]: We're working off of paths here, which contain no useful information if the person is actually a monkey or not. This block will never get called.

**Comment by Krausus** [comment]
Why not just take all the checks you're replacing, move them to a proc, and use that to set `["mobtype"]` to the appropriate fluff string instead of storing a type? It's not actually used for anything else.

File: code/game/machinery/telecomms/logbrowser.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/game/machinery/telecomms/logbrowser.dm]: right.

**Comment by Anticept** [comment]
Hey @Crazylemon64 these devices stopped working. Did they get broken inadvertently? I can't set the network, it always gets stuck on /list and so I can't progress.

**Pull request merged**